app/hub/Views/SupporterView/SupporterView.jsx

```diff
@@ -63,7 +63,7 @@ const SupporterView = (props) => {
 						{_renderButton(isSupporter, true)}
 						<div
 							className="SupporterView__headingCost SupporterView--addSideMargin flex-container align-middle"
-							dangerouslySetInnerHTML={{ __html: t('hub_supporter_price') }}
+							dangerouslySetInnerHTML={{ __html: t('hub_supporter_price', '$') }}
```

Comment:
@zarembsky using `innerHTML` to set the html content will trigger a review complaint form Firefox... can we figure out another way of doing this?

Reply:
We have dangerouslySetInnerHTML all over our code.